packages/wrangler/src/pages/secret/index.ts

```diff
@@ -167,18 +165,22 @@ export const secret = (secretYargs: CommonYargsArgv, subHelp: SubHelp) => {
 			}
 		)
 		.command(
-			"bulk [json]",
+			"bulk [data]",
 			"Bulk upload secrets for a Pages project",
 			(yargs) => {
 				return yargs
-					.positional("json", {
-						describe: `The JSON file of key-value pairs to upload, in form {"key": value, ...}`,
+					.positional("data", {
```

Comment:
Unfortunately this is a breaking change, since yargs supports passing positionals as named args (`--json`). As it happens, we were going to make this change in v4 anyway, so would you be able to rename the arg back to `json` in this PR and open a second PR to the v4 branch that just does this argument rename?

Reply:
Ah thanks for the heads up 🧡